EventFilter/EcalRawToDigi/plugins/alpaka/DeclsForKernels.h

```diff
@@ -0,0 +1,51 @@
+#ifndef EventFilter_EcalRawToDigi_plugins_alpaka_DeclsForKernels_h
+#define EventFilter_EcalRawToDigi_plugins_alpaka_DeclsForKernels_h
+
+#include <optional>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace ecal {
+    namespace raw {
+
+      struct InputDataHost {
+        InputDataHost() = default;
+
+        void initialize(size_t size, size_t nFeds) {
```

Comment:
I think initializing them in the constructor is not possible because the number of FEDs is only known in `produce` after looking at the raw data size.

Reply:
Yes I managed to get it working by deleting the default constructor.